bootstrap/kubeadm/internal/ignition/clc/clc.go

```diff
@@ -96,6 +96,8 @@ systemd:
         Description=kubeadm
         # Run only once. After successful run, this file is moved to /tmp/.
         ConditionPathExists=/etc/kubeadm.yml
+        After=network.target
```

Comment:
I'm on the fence with this condition, since it has been working stable in all environments I have tested it, but it seem generic enough that we could add it, as kubeadm is indeed likely to depend on networking in general.

Generally, such modifications should be applied at the cluster template level, this is why we allow adding extra CLC snippets there, but for the very generic and high level things, we may make an exception.

Reply:
Leaving aside the containerd part, the network target is a must have (at least in baremetal environments and probably in lazy virtual environment), otherwise the control plane or the worker node are very likely to fail when connecting to each other. 

I will remove the containerd part.